README.md

```diff
@@ -1,2 +1,17 @@
-# cla
-The repo where we review and handle CLA requests.
+# Conda Contributor License Agreement
+
+This repo contains and is used to review submissions of Conda Individual Contributor License Agreements.
+
+## Content
+
+Within the `documents` directory, a number of files related to the CLA are shared for the sake
+of transparency. New contributors should complete the Conda Contributor License Agreement
+located [here](https://conda.io/en/latest/contributing.html#conda-contributor-license-agreement).
+
+Google Docs hosted versions of the Conda Individual Contributor License Agreement (2024)
+can be viewed [here](https://docs.google.com/document/d/1R5SO7WSlfecFJAclnw_0Q-lIEGxyAVZ3ucCFh6l4VYE/edit?usp=sharing)
+and [here](https://docs.google.com/document/d/1XMhMS-mQnnhGAOgaCvEPjrn9gWVJjGQoHvW7GdR9ZG8/edit?usp=sharing).
```

Comment:
Is it possible for the copies in this repo to become the official ones? IOW making this text unneeded?